src/wazuh_modules/task_manager/wm_task_manager.c

```diff
@@ -35,18 +36,37 @@ static const char *json_keys[] = {
     [WM_TASK_TASK_ID] = "task_id",
     [WM_TASK_STATUS] = "status",
     [WM_TASK_ERROR] = "error",
-    [WM_TASK_ERROR_DATA] = "data"
+    [WM_TASK_ERROR_DATA] = "data",
+    [WM_TASK_CREATE_TIME] = "create_time",
+    [WM_TASK_LAST_UPDATE_TIME] = "update_time"
+};
+
+static const char *task_statuses[] = {
```

Comment:
This is repeated in the file `wm_task_manager_db.c`. Remove it and add this line in the file `wm_task_manager.h`:
```c
extern const char* task_statuses[];
```

Reply:
done